### ISSUE TITLE
Full routes dataset?

### ISSUE THREAD

**Issue by najwalb** [coleygroup/desp#4]
Hello,

Great work! Do you happen to have the full routes for the test targets you're using (i.e. USPTO-190, and the 2 Pitachio)? The paper contains statistics about the test routes, but the data you shared here only gives the target and the goal starting material.

**Comment by najwalb** [comment]
Ok, I found links to the full routes of USPTO-190 in this [issue](https://github.com/binghong-ml/retro_star/issues/2) in RetroStar's github repo. Here are the links for ease of reference: [train](https://www.dropbox.com/scl/fi/43yw0br07cvdnw9yi1f29/routes_train.pkl?rlkey=zoi85hkd7z7walnd9p5jj0pou&e=1&dl=0), [val](https://www.dropbox.com/scl/fi/2ha93k1wpqs3uylnda9zx/routes_val.pkl?rlkey=7zgiz6zeug9351irxssultapl&e=1&dl=0), [test](https://www.dropbox.com/scl/fi/9pc9hdk5y5vl4gvnua508/routes_test.pkl?rlkey=ib9d8u4wgxz6kwm7cdbt8weq9&e=1&dl=0).

I also made a pull request to retrostar to add the links to their readme. Do I understand correctly that you're using the same routes as RetroStar for USPTO-190? And any chance you can share the pistachio routes?

**Comment by ykevu** [comment]
Hi Najwa, thank you for your interest in our work! We did use the same routes as Retro* for USPTO-190. However, our train/val/test split was different (and can be found at our figshare).

Unfortunately, we cannot share the full routes for the Pistachio sets as they are derived from proprietary reaction data (but we were allowed to release the targets and SMs by NextMove).

**Comment by najwalb** [comment]
Sounds good, thanks for the info!

**Comment by najwalb** [comment]
Sorry, I am reopening the issue just to confirm I got the right files from figshare: these would be under [reaction_templates_with_splits](https://figshare.com/articles/preprint/Pre-trained_models_and_data_for_Double-Ended_Synthesis_Planning_DESP_/25956076?file=47534525) right?

I noticed the files contain single reactions instead of complete routes, and there is no split for test data.

**Comment by ykevu** [comment]
Yes, those are the right files. The reactions were never fully converted into a route dataset and saved, as we only trained NNs using individual reactions within routes. If you take a look at [this script](https://github.com/coleygroup/desp/blob/main/processing/04_extract_fwd_training.py) for example, routes are being constructed on the fly using the reaction data.

There is no test data because we used USPTO-190 and the Pistachio benchmarks for testing.

Hope this helps!

**Comment by najwalb** [comment]
Yes it does thanks a lot! One last question :) 

How did you get access to pistachio? I emailed nextmove at info@nextmovesoftware.com, but haven't heard back from them yet.

**Comment by ykevu** [comment]
I believe MIT as a whole has access through an academic license; the email you listed should be the correct point of contact!

**Comment by najwalb** [comment]
Cool thanks again for all the info :)
